bookkeeper-dist/src/main/resources/LICENSE-all.bin.txt

```diff
@@ -202,507 +202,265 @@
-   computer software. (as that term is defined at 48 C.F.R. ?
-   252.227-7014(a)(1)) and .commercial computer software
-   documentation. as such terms are used in 48 C.F.R. 12.212
-   (Sept. 1995). Consistent with 48 C.F.R. 12.212 and 48
-   C.F.R. 227.7202-1 through 227.7202-4 (June 1995), all
-   U.S. Government End Users acquire Covered Software with only those
-   rights set forth herein. This U.S. Government Rights clause is in
-   lieu of, and supersedes, any other FAR, DFAR, or other clause or
-   provision that addresses Government rights in computer software
-   under this License.
-
-9. MISCELLANEOUS.
-
-   This License represents the complete agreement concerning subject
-   matter hereof. If any provision of this License is held to be
-   unenforceable, such provision shall be reformed only to the extent
-   necessary to make it enforceable. This License shall be governed by
-   the law of the jurisdiction specified in a notice contained within
-   the Original Software (except to the extent applicable law, if any,
-   provides otherwise), excluding such jurisdiction.s conflict-of-law
-   provisions. Any litigation relating to this License shall be
-   subject to the jurisdiction of the courts located in the
-   jurisdiction and venue specified in a notice contained within the
-   Original Software, with the losing party responsible for costs,
-   including, without limitation, court costs and reasonable
-   attorneys. fees and expenses. The application of the United Nations
-   Convention on Contracts for the International Sale of Goods is
-   expressly excluded. Any law or regulation which provides that the
-   language of a contract shall be construed against the drafter shall
-   not apply to this License. You agree that You alone are responsible
-   for compliance with the United States export administration
-   regulations (and the export control laws and regulation of any
-   other countries) when You use, distribute or otherwise make
-   available any Covered Software.
-
-10. RESPONSIBILITY FOR CLAIMS.
-
-   As between Initial Developer and the Contributors, each party is
-   responsible for claims and damages arising, directly or indirectly,
-   out of its utilization of rights under this License and You agree
-   to work with Initial Developer and Contributors to distribute such
-   responsibility on an equitable basis. Nothing herein is intended or
-   shall be deemed to constitute any admission of liability.
+lib/io.netty-netty-all-4.1.12.Final.jar contains a modified portion of and optionally depends on 'Protocol Buffers', Google's data
+interchange format, which can be obtained at:
 
+  * LICENSE:
+    * deps/netty-4.1.12.Final/LICENSE.protobuf.txt (New BSD License)
+  * HOMEPAGE:
+    * https://github.com/google/protobuf
+
+------------------------------------------------------------------------------------
+This product bundles the JSR-305 reference implementation, which is available under
```

Comment:
we should exclude this jar since it is compliation dependency not runtime dependency.

Reply:
Are you sure? I was under the same impression, but when I looked at the source, the annotations have runtime retention. If we can get away with removing it, I'm all for that.